src/jit/lsra.cpp

```diff
@@ -6655,7 +6655,7 @@ void LinearScan::processBlockStartLocations(BasicBlock* currentBlock, bool alloc
     VarToRegMap inVarToRegMap     = getInVarToRegMap(currentBlock->bbNum);
     bool        hasCriticalInEdge = blockInfo[currentBlock->bbNum].hasCriticalInEdge;
 
-    VARSET_TP VARSET_INIT_NOCOPY(liveIn, currentBlock->bbLiveIn);
```

Comment:
It is a bit more complex. What was write here meant: there are two varsets, chose one and read data from that one. The problem is that it is hard to protect live_in set from a modification, moreover it will have different effects in cases of short/long sets. So I rewrote this part with pointers, now it should be better.